.github/CHANGELOG.md

```diff
@@ -328,8 +368,6 @@
   beyond VQE. Use of `VQECost` is still possible but will result in a deprecation warning.
   [(#913)](https://github.com/PennyLaneAI/pennylane/pull/913)
 
-<h3>Documentation</h3>
-
 <h3>Bug fixes</h3>
 
 * The `default.qubit.tf` device is updated to handle TensorFlow objects (e.g.,
```

Comment:
The inverse tricks aren't really "Bug fixes". More "Improvements"?

Reply:
This one?

> The `default.qubit.tf` device is updated to handle TensorFlow objects (e.g., `tf.Variable`) as gate parameters correctly when using the `MultiRZ` and `CRot` operations.

Previously, this would error if you passed a tensorflow variable as the gate parameter, so I would class this one as a bug fix 🤔

Reply:
no, the one with the word "tricks" in it :)

Reply:
Oh! Right.

I think this one is badly worded. It was a bug fix, in that we _thought_ inverses were being applied via the permutation methods, but there was an error in the if statement, causing them to fallback to the einsum `_apply_unitary`.

Reply:
Reworded this one!